ci/build/build-vscode.sh

```diff
@@ -6,12 +6,25 @@ set -euo pipefail
 # MINIFY controls whether a minified version of vscode is built.
 MINIFY=${MINIFY-true}
 
+delete-bin-script() {
+  rm "lib/vscode-reh-web-linux-x64/bin/$1"
+}
+
+copy-bin-script() {
+  local script="$1"
+  local dest="lib/vscode-reh-web-linux-x64/bin/$script"
+  cp "lib/vscode/resources/server/bin/$script" "$dest"
+  sed -i "s/@@VERSION@@/$(vscode_version)/g" "$dest"
```

Comment:
reminder: may want to look into making sure it works on both macOS and Linux: https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux

Reply:
> The command (note the lack of space between -i and '' and the -e to make it work on new versions of Mac and on GNU):

```shell
sed -i'.original' -e 's/old_link/new_link/g' *
```